react/src/Containers/RestaurantsIndexContainer.js

```diff
@@ -25,13 +25,15 @@ class RestaurantsIndexContainer extends Component {
       return(
```

Comment:
minor point, but I like to make the local variable more descriptive (`restaurant` instead of `r`)